index.html

```diff
@@ -762,9 +762,311 @@
         <!-- ************ -->
 
         <section id="class-3">
-          <h2>Third Class: Jeff</h2>
+          <h2>Third Class</h2>
+          <p>Jeff O'Leary</p>
         </section>
+        <section>
+          <h4>Agenda</h4>
+          <p class="fragment">Navigation</p>
+          <p class="fragment">Break time!</p>
+          <p class="fragment">Auto Layout</p>
+        </section>
+        <section>
+          <h4>Goals</h4>          
+          <p class="fragment">What is a navigation stack</p>
+          <p class="fragment">Who coordinates and controls navigation</p>
+          <p class="fragment">How to navigate throughout an app</p>
+          <p class="fragment">What is Auto Layout</p>
+          <p class="fragment">What is the benefit of Auto Layout</p>
+          <p class="fragment">How do I use Auto Layout</p>
+            <aside class="notes">
+                The goal of this class is by the end of the night, you will understand...
+            </aside>
+        </section>
+          <section>
+          <h4>Goals</h4>
+          <p class="fragment">If these goals aren't met, I'm doing something wrong.  Please help me be a better teacher, ask any and all questions!</p>
+          <br/>
+          <p class="fragment">Any follow up questions or clarification, feel to use the anonymous email provided - gdi.ios.anonymous.question@gmail.com<br/>@GDI.Question!</p>
+          <br/>
+          <p class="fragment">my email - joleary1987@gmail.com</p>
+        </section>
+          <section>
+              <p>Xcode has tons of panels and toolbars </p>
+              <p class="fragment">Here is a great cheat sheet reference that provides a thorough tour of Xcode in just a few slides</p>
+              <a href="http://bit.ly/1BcjmI5" class="fragment">http://bit.ly/1BcjmI5</a>
+              <p class="fragment">(will put it in Slack chat as well)</p>
+          </section>
+        <section>
+          <h2>Navigation</h2>
+            <p class="fragment">Navigation Interface</p>
+            <p class="fragment">UINavigationController</p>
+            <p class="fragment">StoryBoard Segues</p>
```

Comment:
We already introduced storyboards and we didn't go over creating xibs for
view controllers so I figured I'd go over all navigation options.
Especially with segues being the easiest to drag and drop in the storyboard

On Sunday, June 14, 2015, Elliot Schrock <notifications@github.com> wrote:

> In index.html
> <https://github.com/gdiboston/gdi-intro-ios/pull/8#discussion_r32386448>:
>
> > +          <br/>
> > +          <p class="fragment">Any follow up questions or clarification, feel to use the anonymous email provided - gdi.ios.anonymous.question@gmail.com <javascript:_e(%7B%7D,'cvml','gdi.ios.anonymous.question@gmail.com');><br/>@GDI.Question!</p>
> > +          <br/>
> > +          <p class="fragment">my email - joleary1987@gmail.com <javascript:_e(%7B%7D,'cvml','joleary1987@gmail.com');></p>
> > +        </section>
> > +          <section>
> > +              <p>Xcode has tons of panels and toolbars </p>
> > +              <p class="fragment">Here is a great cheat sheet reference that provides a thorough tour of Xcode in just a few slides</p>
> > +              <a href="http://bit.ly/1BcjmI5" class="fragment">http://bit.ly/1BcjmI5</a>
> > +              <p class="fragment">(will put it in Slack chat as well)</p>
> > +          </section>
> > +        <section>
> > +          <h2>Navigation</h2>
> > +            <p class="fragment">Navigation Interface</p>
> > +            <p class="fragment">UINavigationController</p>
> > +            <p class="fragment">StoryBoard Segues</p>
>
> Do we want to do this? Or are we doing XIBs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdiboston/gdi-intro-ios/pull/8/files#r32386448>.
>


-- 
Best Regards,
Jeff O'Leary


[image: MeYou Health][image: FB] <https://www.facebook.com/meyouhealth>[image:
T] <http://twitter.com/#!/meyouhealth>[image: RSS]
<http://www.meyouhealth.com/blog/>[image: YT]
<http://www.youtube.com/user/meyouhealthvideos>[image: F]
<http://www.flickr.com/photos/meyouhealth>
*Jeff O'Leary*iOS Engineer
Cell: 857-268-2939
MeYou Health - the social well-being companywww.meyouhealth.com
jeff.oleary@meyouhealth.com
MeYou Health is a Healthways, Inc. company